### ISSUE TITLE
"Select All" is really needed

### ISSUE THREAD

**Issue by 949mac** [Silvanite/novafieldcheckboxes#34]
Hello, 

It would be terrific if you give us an option for "Select All". It would greatly reduce my medical costs to US doctors to treat my carpal tunnel from clicking my poorly designed resources. 

[image](https://user-images.githubusercontent.com/23151962/83080163-5dc19c80-a032-11ea-8d28-2c2e29e2d1a9.png)

All jokes aside, thank you very much!

**Comment by perle-rare** [comment]
Thanks for this proposal, I need the same functionality

**Comment by m2de** [comment]
Agreed, this would be a good addition, already marked as desired enhancement in #20 . **Check / Uncheck all**
